ITISFoundation/osparc-simcore
# ⬆️ Upgrade to traefik v3 (2nd trial)

This reverts commit b1416b4b641cbbc4a9c6614cb13851b5fe37d6bf.
This needs to be merged together with https://github.com/ITISFoundation/osparc-ops-environments/pull/743


## What do these changes do?
Upgrade traefik major version to v3


## Related issue/s
https://github.com/ITISFoundation/osparc-ops-environments/pull/743



## How to test
We need to check whether on both master-deployments legacy and v2 dynamic services start, there was an issue the last time this was merged. 


## Dev-ops checklist
